alert-common/src/main/java/com/synopsys/integration/alert/common/rest/model/MultiFieldModel.java

```diff
@@ -24,14 +24,13 @@
 
 import java.util.List;
 
-public class MultiFieldModel extends AlertSerializableModel {
-    private List<FieldModel> fieldModels;
-
-    public MultiFieldModel(List<FieldModel> fieldModels) {
-        this.fieldModels = fieldModels;
+public class MultiFieldModel extends MultiResponseModel<FieldModel> {
+    public MultiFieldModel(final List<FieldModel> models) {
```

Comment:
I'm shocked to see a `final` here.

Reply:
Why constructors, but not elsewhere? That seems inconsistent with the libraries which definitely modify things like headers and requestBodies in constructors.

Reply:
I can address the final, but I don't understand the comment about the constructors, I'll follow up with you.